templates/web/base/admin/reports/_edit_main.html

```diff
@@ -64,6 +64,14 @@
     <input type="submit" class="btn" name="mark_sent" value="[% loc('Mark as sent') %]">
   [% END %]
 </li>
+[% IF NOT problem.whensent %]
```

Comment:
We probably want to restrict this to superusers, might be hard to explain otherwise.